asteroid/masknn/blocks.py

```diff
@@ -50,6 +50,50 @@ def forward(self, x):
         return res_out, skip_out
 
 
+class SeparableDilatedConv2DBlock(nn.Module):
+    """One dimensional convolutional block, as proposed in [1] without skip
+        output. As used in the two step approach [2]. This block uses the
+        groupnorm across features and also produces always a padded output.
+
+    Args:
+        in_chan (int): Number of input channels.
+        hid_chan (int): Number of hidden channels in the depth-wise
+            convolution.
+        kernel_size (int): Size of the depth-wise convolutional kernel.
+        dilation (int): Dilation of the depth-wise convolution.
+
+    References:
+        [1]: "Conv-TasNet: Surpassing ideal time-frequency magnitude masking
+             for speech separation" TASLP 2019 Yi Luo, Nima Mesgarani
+             https://arxiv.org/abs/1809.07454
```

Comment:
These blocks were introduced in the v1, we can note that without skip connection it corresponds to them
https://arxiv.org/abs/1809.07454v1